tests/playwright/sanity/modules/styleguide/styleguide-1.test.ts

```diff
@@ -144,8 +144,7 @@ test.describe( 'Styleguide Preview tests @styleguide_image_link', () => {
 
 	test( 'Switching between tabs makes relevant area visible', async ( { page, apiRequests }, testInfo ) => {
 		// Arrange.
-		const { editor } = await getInSettingsTab( page, testInfo, apiRequests, 'Global Colors', true );
-		await page.waitForTimeout( 2000 );
```

Comment:
No need to wait. In other places where we use `getInSettingsTab()`, we don't wait. It only increases the test execution time.